### PR TITLE
Add SolverName Attribute to MOI Interface

### PR DESCRIPTION
**Pull request by ccoffrin**

### CODE REVIEW

File: test/MOI_wrapper.jl [hunk 1]
```diff
@@ -20,6 +20,10 @@ const optimizer = Ipopt.Optimizer(print_level=0, fixed_variable_treatment="make_
 const config = MOIT.TestConfig(atol=1e-4, rtol=1e-4,
                                optimal_status=MOI.LOCALLY_SOLVED)
 
+@testset "MOI API tests" begin
+  @test MOI.get(optimizer, MOI.SolverName()) == "Ipopt"
```
Comment (blegat) [inline test/MOI_wrapper.jl]: 4-space indent

File: test/MOI_wrapper.jl [hunk 2]
```diff
@@ -20,6 +20,10 @@ const optimizer = Ipopt.Optimizer(print_level=0, fixed_variable_treatment="make_
 const config = MOIT.TestConfig(atol=1e-4, rtol=1e-4,
                                optimal_status=MOI.LOCALLY_SOLVED)
 
+@testset "MOI API tests" begin
```
Comment (blegat) [inline test/MOI_wrapper.jl]: The test set name should be `SolverName` according to the testing guideline:
http://www.juliaopt.org/MathOptInterface.jl/dev/apimanual/#Testing-guideline-1

File: test/MOI_wrapper.jl (hunk 1 above)
Comment (ccoffrin) [reply test/MOI_wrapper.jl]: Note that, indentation spacing is not consistent in the rest of this file.

File: test/MOI_wrapper.jl (hunk 2 above)
Comment (ccoffrin) [reply test/MOI_wrapper.jl]: I stole this from Gurobi.jl, you might want to post an issue there, https://github.com/JuliaOpt/Gurobi.jl/blob/master/test/MOI_wrapper.jl#L82

**Pull request merged**